test/fail_compilation/b6226.d

```diff
@@ -0,0 +1,22 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/b6226.d(13): Error: cannot implicitly convert expression `400000000` of type `int` to `ubyte`
```

Comment:
This error message is incorrect. `c` undergoes integral promotions to become an `int`. But that promotion cannot produce 4000000000.

Reply:
Just a drive-by cause I saw this in my email, but the expression says 400000000, not 4000000000 (with commas, that's 400,000,000 and 4,000,000,000 respectively).

Reply:
I just put a bunch of 0's. The number is irrelevant to this case.